src/components/relaying-education-series/RelaySeriesStepper/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Card, IconButton, SvgIcon } from '@mui/material'
+import Navigator from '@/components/relaying-education-series/Navigator'
+import useEducationSeriesStepper, {
+  type EducationSeriesStepperProps,
+} from '@/components/relaying-education-series/RelaySeriesStepper/useEducationSeriesStepper'
+import ProgressBar from '@/components/common/ProgressBar'
+import CloseIcon from '@/public/images/common/close.svg'
+import IndexNumber from '@/components/relaying-education-series/IndexNumber'
+import css from './styles.module.css'
+
+const RelaySeriesStepper = (props: EducationSeriesStepperProps) => {
+  const { onBack, onNext, activeStep, setStep, onClose, progress } = useEducationSeriesStepper({ ...props })
+  const { steps } = props
+  const currentStep = steps[activeStep]
+
+  return (
+    <Box className={css.wrapper}>
+      <Card className={css.infoCard}>
+        <ProgressBar value={progress} />
+        <Box display="flex" alignItems="center" gap="12px">
+          <IndexNumber value={activeStep + 1} />
+          <h1>{currentStep.title}</h1>
+          <span style={{ flex: '1' }} />
+          <IconButton onClick={onClose}>
+            <SvgIcon component={CloseIcon} inheritViewBox fontSize="small" />
```

Comment:
This does not match the colour of other modals:

![image](https://user-images.githubusercontent.com/20442784/226899683-9345d6c4-6985-4ebc-83b6-7001328676f3.png)

![image](https://user-images.githubusercontent.com/20442784/226899709-e0d260bd-7ce2-4ccb-8f46-11a6372b7288.png)

Reply:
I've updated to the color present in the Figma designs `primary.main`